         
时间： 2021-12-09 20:20:59.6543         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 20:20:59
异常类型：NullReferenceException
异常消息：未将对象引用设置到对象的实例。
异常信息：   在 NanXingPengMaServices.Form1.Form1_Load(Object sender, EventArgs e) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Form1.cs:行号 72
   在 System.Windows.Forms.Form.OnLoad(EventArgs e)
   在 System.Windows.Forms.Form.OnCreateControl()
   在 System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   在 System.Windows.Forms.Control.CreateControl()
   在 System.Windows.Forms.Control.WmShowWindow(Message& m)
   在 System.Windows.Forms.Control.WndProc(Message& m)
   在 System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   在 System.Windows.Forms.Form.WmShowWindow(Message& m)
   在 System.Windows.Forms.Form.WndProc(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   在 System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
         
堆栈：  ThreadContext.OnThreadException => Program.Application_ThreadException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:04:58.9326         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:04:58
Application UnhandledException:String 引用没有设置为 String 的实例。
参数名: s;
堆栈信息:   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 361
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 250
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 228
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Encoding.GetBytes => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:07:45.4896         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:07:45
Application UnhandledException:值不能为 null。
参数名: String;
堆栈信息:   在 System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   在 System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   在 System.Int32.Parse(String s)
   在 NanXingPengMaServices.Utils.TiShengJiHelper.ParseMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TiShengJiHelper.cs:行号 127
   在 NanXingPengMaServices.Utils.TiShengJiHelper.ReceiveTcpMsg(String strMsg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TiShengJiHelper.cs:行号 68
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ReceiveCallback(IAsyncResult ar) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 320
   在 System.Net.LazyAsyncResult.Complete(IntPtr userToken)
   在 System.Net.ContextAwareResult.CompleteCallback(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Net.ContextAwareResult.Complete(IntPtr userToken)
   在 System.Net.LazyAsyncResult.ProtectedInvokeCallback(Object result, IntPtr userToken)
   在 System.Net.Sockets.BaseOverlappedAsyncResult.CompletionPortCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* nativeOverlapped)
   在 System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pOVERLAP)         
堆栈：  Number.StringToNumber => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:09:06.5561         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:09:06
Application UnhandledException:更新条目时出错。有关详细信息，请参阅内部异常。;
堆栈信息:   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 NanXingPengMaServices.Utils.TiShengJiHelper.ParseMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TiShengJiHelper.cs:行号 145
   在 NanXingPengMaServices.Utils.TiShengJiHelper.ReceiveTcpMsg(String strMsg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TiShengJiHelper.cs:行号 68
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ReceiveCallback(IAsyncResult ar) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 320
   在 System.Net.LazyAsyncResult.Complete(IntPtr userToken)
   在 System.Net.ContextAwareResult.CompleteCallback(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Net.ContextAwareResult.Complete(IntPtr userToken)
   在 System.Net.LazyAsyncResult.ProtectedInvokeCallback(Object result, IntPtr userToken)
   在 System.Net.Sockets.BaseOverlappedAsyncResult.CompletionPortCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* nativeOverlapped)
   在 System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pOVERLAP)         
堆栈：  InternalContext.SaveChanges => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:17:55.7753         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:17:55
Application UnhandledException:String 引用没有设置为 String 的实例。
参数名: s;
堆栈信息:   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection()
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive()
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Encoding.GetBytes => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:32:03.8704         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:32:03
Application UnhandledException:String 引用没有设置为 String 的实例。
参数名: s;
堆栈信息:   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 361
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 250
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 228
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Encoding.GetBytes => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:33:59.0152         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:33:58
Application UnhandledException:String 引用没有设置为 String 的实例。
参数名: s;
堆栈信息:   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 361
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 250
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 228
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Encoding.GetBytes => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:36:46.9741         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息： System.ArgumentNullException: String 引用没有设置为 String 的实例。
参数名: s
   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 363         
堆栈：  SocketClientManager.ClientAuthenticationAndReconnection => SocketClientManager.SendMsg => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:36:58.2609         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:36:58
Application UnhandledException:同一个套接字上正在进行另一个异步操作时，不能调用 BeginConnect。;
堆栈信息:   在 System.Net.Sockets.Socket.BeginConnect(EndPoint remoteEP, AsyncCallback callback, Object state)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 278
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 228
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Socket.BeginConnect => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:38:14.0653         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:38:13
异常类型：IndexOutOfRangeException
异常消息：索引超出了数组界限。
异常信息：   在 NanXingPengMaServices.Form1.UpdateUI(Int32 type, String str, TiShengJiState[] ts) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Form1.cs:行号 733
         
堆栈：  ThreadContext.OnThreadException => Program.Application_ThreadException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:39:02.1431         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息： System.ArgumentNullException: String 引用没有设置为 String 的实例。
参数名: s
   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 363         
堆栈：  SocketClientManager.ClientAuthenticationAndReconnection => SocketClientManager.SendMsg => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-09 22:39:13.4370         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/9 22:39:13
Application UnhandledException:同一个套接字上正在进行另一个异步操作时，不能调用 BeginConnect。;
堆栈信息:   在 System.Net.Sockets.Socket.BeginConnect(EndPoint remoteEP, AsyncCallback callback, Object state)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 278
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 228
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Socket.BeginConnect => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
